cmd/host/main.go

```diff
@@ -18,28 +18,48 @@ import (
 )
 
 var (
-	debug     bool
-	mtu       int
-	endpoints arrayFlags
+	debug        bool
+	mtu          int
+	endpoints    arrayFlags
+	vpnkitSocket string
+	qemuSocket   string
 )
 
 func main() {
 	flag.Var(&endpoints, "listen", fmt.Sprintf("url where the tap send packets (default %s)", transport.DefaultURL))
 	flag.BoolVar(&debug, "debug", false, "debug")
 	flag.IntVar(&mtu, "mtu", 1500, "mtu")
+	flag.StringVar(&vpnkitSocket, "listen-vpnkit", "", "VPNKit socket to be used by Hyperkit")
+	flag.StringVar(&qemuSocket, "listen-qemu", "", "Socket to be used by Qemu")
 	flag.Parse()
 
+	if debug {
+		log.SetLevel(log.DebugLevel)
+	}
+
 	if len(endpoints) == 0 {
 		endpoints = append(endpoints, transport.DefaultURL)
 	}
 
+	if vpnkitSocket != "" && qemuSocket != "" {
+		log.Fatal("cannot use qemu and vpnkit protocol at the same time")
```

Comment:
One problem with this approach is that any defer'd functions do not get executed, like socket clean up, etc.  We should consider either moving this into an inline go func or better yet, have main basically just do cli and call out to the various functions and exit gracefully.